### ISSUE TITLE
Attestation BLS verification is bypassed

### ISSUE THREAD

**Issue by paulhauner** [sigp/lighthouse#3]
https://github.com/sigp/rust_beacon_chain/blob/6f0bbd47fac20652472c1ff5799aac367af5d929/src/state/transition/attestations.rs#L30-L37

This is blocked by #1.

**Comment by paulhauner** [comment]
No longer relevant -- v2.1 spec requires a full re-write
